wasmapi/go/map.go

```diff
@@ -60,7 +60,7 @@ type MapType uint32
 const (
 	UnspecifiedMap MapType = iota
 	Hash
-	Array
+	ArrayMap
```

Comment:
In this commit and even while PR there is no other mention of `ArrayMap`. Is this used anywhere?